org/polypheny/prism/value.proto

```diff
@@ -288,3 +288,55 @@ message ProtoFile {
   // The content of the file in binary.
   bytes binary = 1;
 }
+
+/*
+ * This message represents a node in the labeled property graph model.
+ * Apart form a unique identifier and a name, a set of entries and list are used to represent properties and labels.
+ */
+message ProtoNode {
+  // The system internal unique identifier for that node.
+  string id = 1;
+  // The name of the node as specified on creation.
+  string name = 2;
+  // A map of key-value pairs of proto values. These are used as a map to store properties with their corresponding names.
+  repeated ProtoEntry properties = 3;
```

Comment:
Can you use a `map` instead of repeated to ensure that there are no duplicate keys?

Reply:
Absolutely. I will adjust this and push the update.